README.md

```diff
@@ -1,4 +1,6 @@
 # LFT2
+![Unittest](https://github.com/icon-project/LFT2/workflows/Unittest/badge.svg)
+![IntegrationTest](https://github.com/icon-project/LFT2/workflows/IntegrationTest/badge.svg)
```

Comment:
I think those badges would be better: 
- show master branch status

See: (example-using-the-branch-parameter)[https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#example-using-the-branch-parameter]